src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/cache/preemptive/PreemptiveCacheRefresher.kt

```diff
@@ -43,9 +54,9 @@ class PreemptiveCacheRefresher(
 
   fun haveFlywayMigrationsFinished(): Boolean {
     return try {
-      flyway.validate()
-      true
-    } catch (_: FlywayValidateException) {
+      return flyway.info().pending().none { !it.script.startsWith("R__") }
```

Comment:
I'm assuming this is something to do with the repeatable migrations? Might be worth breaking this out into a named function like `allPendingNonRepeatableMigrationsAreComplete()`